examples/ros/RExFeed.jl

```diff
@@ -42,69 +49,161 @@ using DocStringExtensions
 # end
 
 # Set this
-global max_samples = 100
 global dfg_datafolder = "/tmp/rex"
 
+"""
+    $TYPEDEF
+Quick placeholder for the system state - we're going to use timestamps to align all the data.
+"""
+mutable struct SystemState
```

Comment:
Great, this seems to be one of those standard SLAM things that keep coming up.  At some point we will consolidate.  There's a really old one in RoME called SLAMWrapper.  Guess a good name for this is the first important thing.  I recently created a new one (just `Dict{Symbol, Any}`) called `dashboard` -- pretty boring.